pkg/cmd/server/server.go

```diff
@@ -543,7 +558,12 @@ func (s *server) initRestic(location *api.BackupStorageLocation) error {
 	return nil
 }
 
-func (s *server) runControllers(config *api.Config, defaultBackupLocation *api.BackupStorageLocation) error {
+func (s *server) runControllers(
+	config *api.Config,
+	defaultBackupLocation *api.BackupStorageLocation,
+	defaultVolumeSnapshotLocations map[string]string,
```

Comment:
I noticed that in another PR we have the controller expecting a `map[string]string` but since we are doing CLI validation anyway, at this point, I can pass it a `[]*api.VolumeSnapshotLocation`

Reply:
How about a `map[string]*api.VolumeSnapshotLocation`, where keys are provider names?  I think that's probably the most useful format to have it in.  cc @carlisia for input

Reply:
Thinking...

Reply:
Having a `[]*api.VolumeSnapshotLocation` will work perfectly, so let's go with that.